solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -1573,13 +1573,20 @@ public void open() {
    */
   @Override
   public void close() {
-    int count = refCount.decrementAndGet();
-    if (count > 0) return; // close is called often, and only actually closes if nothing is using it.
```

Comment:
I kind of like the comment here explaining that close is called often and I'll be a little sad about losing it.

Reply:
The javadocs on this method are excellent though, so it seemed redundant